src/Quartz/Impl/StdSchedulerFactory.cs

```diff
@@ -508,11 +516,14 @@ protected virtual void ValidateConfiguration()
 
             Type tpType = loadHelper.LoadType(cfg.GetStringProperty(PropertyThreadPoolType)) ?? typeof(SimpleThreadPool);
 
+#if WINDOWS_THREADPOOL  // TODO (NetCore Port) - We can't use the ClrThreadPool on .NET Core because cross-plat threadpool implementations don't expose all the APIs we need
+                        // Use the SimpleThreadPool there for now and in the future create a new async-friendly threadpool that will work cross-plat.
             if (tpType == typeof(SimpleThreadPool))
```

Comment:
Sounds promising. Thread priorities are a bit of an oddity in a way. They do share the same priority in pool (the original implementation) but jobs themselves were able to boost priority when starting (setting current thread to higher priority). Thread pool resets worker's priority back to pool's default after job completes.

Quartz also has notion of trigger priority which determines the order how triggers are queued for execution when multiple triggers are scheduled to run in same retrieved batch. This affects execution order but not the resources allowed for the job running.

In my opinion, boosting thread/task priority can be left as an enhancement for later. In async/await world there might not be many who run own actual Thread instances beside Quartz's threads inside same process/appdomain - this is were the prioritization would have an effect. Per task priority of course is an interesting ability but yet again can be different between core and desktop.